### PR TITLE
bump puppeteer version to v3.0

### PR DESCRIPTION
**Pull request by PeterNgTr**
## Motivation/Description of the PR
- Bump puppeteer to v3 which is supporting running tests with firefox instead of using puppeteer-firefox (available since v2 but we haven't got chance to update it).

Applicable helpers:

- [x] Puppeteer

## Type of change

- [ ] :fire: Breaking changes
- [ ] :rocket: New functionality
- [ ] :bug: Bug fix
- [ ] :clipboard: Documentation changes/updates
- [ ] :hotsprings: Hot fix
- [ ] :hammer: Markdown files fix - not related to source code
- [ ] :nail_care: Polish code

## Checklist:

- [ ] Tests have been added
- [ ] Documentation has been added (Run `npm run docs`)
- [x] Lint checking (Run `npm run lint`)
- [x] Local tests are passed (Run `npm test`)

## Notes:
- Those tests are skipped due to bug https://github.com/puppeteer/puppeteer/issues/5543

### CODE REVIEW

File: test/helper/webapi.js [hunk 1]
```diff
@@ -648,7 +648,8 @@ module.exports.tests = function () {
     });
   });
 
-  describe('#attachFile', () => {
+  describe.skip('#attachFile', () => {
+    // Those tests are skipped due to bug https://github.com/puppeteer/puppeteer/issues/5543
```
Comment (elukoyanov) [inline test/helper/webapi.js:652]: could you mark this as ` // TODO: ....` ?

possibly, we can create issue with this problem, to see it
and type link on it `// TODO: #xxxx bla bla....` or `// TODO: https://github.com/Codeception/CodeceptJS/issues/xxxx bla bla...`

File: test/helper/Puppeteer_test.js [hunk 2]
```diff
@@ -17,7 +17,9 @@ let page;
 let FS;
 const siteUrl = TestHelper.siteUrl();
 
-describe('Puppeteer - BasicAuth', () => {
+describe('Puppeteer - BasicAuth', function () {
+  this.timeout(35000);
```
Comment (elukoyanov) [inline test/helper/Puppeteer_test.js:21]: why so much?

File: test/helper/webapi.js [hunk 3]
```diff
@@ -648,7 +648,8 @@ module.exports.tests = function () {
     });
   });
 
-  describe('#attachFile', () => {
+  describe.skip('#attachFile', () => {
```
Comment (DavertMik) [inline test/helper/webapi.js:651]: ups, this is going to skip this test for ALL helpers!
please check webapi.js there are examples how to skip tests for exactly one helper

**Comment by DavertMik** [comment]
Closed in favor of #2355 
Btw, Puppeteer actually fixed downloads issue, so since 3.0.1 they are perfect!

**Pull request closed without merging**